src/components/ec-currency-input/ec-currency-input.vue

```diff
@@ -200,6 +208,16 @@ export default {
     currenciesItems() {
       return this.currencies.map(currency => ({ text: currency, value: currency, id: currency }));
     },
+    currenciesTooltip() {
```

Comment:
Passing an empty object into the prop will show an empty tooltip, we should be rather checking that the content is present:

```
currenciesTooltipOptions() {
  let { content, placement = 'top' } = this.currenciesTooltip || {};
  if (!content) {
    return null;
  } else {
    return { content, placement };
  }
},
```

Reply:
Much better. Updated. Thanks!